src/coreclr/jit/emitloongarch64.cpp

```diff
@@ -2620,24 +2620,14 @@ unsigned emitter::emitOutputCall(insGroup* ig, BYTE* dst, instrDesc* id, code_t
 
     dst += 4;
 
-    // update volatile regs within emitThisGCrefRegs and emitThisByrefRegs.
```

Comment:
Why is this no longer needed? This method would also kill them for other type (e.g. `GCT_GCREF` would kill the `gcrefRegs` from `emitThisByrefRegs` and vice-versa if `GCT_BYREF`,  it will kill `byrefRegs` from `emitThisGCrefRegs`?

Reply:
This is only for liking MIPS64 where there is a branch delay slot but there is no branch delay on LoongArch64.
It's an old code which from the MIPS64 when I did on 2019.